PsfLauncher/main.cpp

```diff
@@ -1,352 +1,352 @@
-{
-    return ((str.length() >= suffix.length()) && (str.substr(str.length() - suffix.length()) == suffix));
-}
-
-void LogApplicationAndProcessesCollection()
-{
-    auto configRoot = PSFQueryConfigRoot();
-
-    if (auto applications = configRoot->as_object().try_get("applications"))
-    {
-        for (auto& applicationsConfig : applications->as_array())
-        {
-            auto exeStr = applicationsConfig.as_object().try_get("executable")->as_string().wide();
-            auto idStr = applicationsConfig.as_object().try_get("id")->as_string().wide();
-            TraceLoggingWrite(
-                g_Log_ETW_ComponentProvider,
-                "ApplicationsConfigdata",
-                TraceLoggingWideString(exeStr, "applications_executable"),
-                TraceLoggingWideString(idStr, "applications_id"),
-                TraceLoggingBoolean(TRUE, "UTCReplace_AppSessionGuid"),
-                TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
-                TraceLoggingKeyword(MICROSOFT_KEYWORD_CRITICAL_DATA));
-        }
-    }
-
-    if (auto processes = configRoot->as_object().try_get("processes"))
-    {
-        for (auto& processConfig : processes->as_array())
-        {
-            auto exeStr = processConfig.as_object().get("executable").as_string().wide();
-            TraceLoggingWrite(
-                g_Log_ETW_ComponentProvider,
-                "ProcessesExecutableConfigdata",
-                TraceLoggingWideString(exeStr, "processes_executable"),
-                TraceLoggingBoolean(TRUE, "UTCReplace_AppSessionGuid"),
-                TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
-                TraceLoggingKeyword(MICROSOFT_KEYWORD_CRITICAL_DATA));
-
-            if (auto fixups = processConfig.as_object().try_get("fixups"))
-            {
-                for (auto& fixupConfig : fixups->as_array())
-                {
-                    auto dllStr = fixupConfig.as_object().try_get("dll")->as_string().wide();
-                    TraceLoggingWrite(
-                        g_Log_ETW_ComponentProvider,
-                        "ProcessesFixUpConfigdata",
-                        TraceLoggingWideString(dllStr, "processes_fixups"),
-                        TraceLoggingBoolean(TRUE, "UTCReplace_AppSessionGuid"),
-                        TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
-                        TraceLoggingKeyword(MICROSOFT_KEYWORD_CRITICAL_DATA));
-                }
-            }
-        }
-    }
-}
-
-bool IsCurrentOSRS2OrGreater()
-{
-    OSVERSIONINFOEXW osvi = { sizeof(osvi), 0, 0, 0, 0, {0}, 0, 0 };
-    DWORDLONG const dwlConditionMask = VerSetConditionMask(0, VER_BUILDNUMBER, VER_GREATER_EQUAL);
-    osvi.dwBuildNumber = 15063;
-
-    return VerifyVersionInfoW(&osvi, VER_BUILDNUMBER, dwlConditionMask);
+//-------------------------------------------------------------------------------------------------------

```

Comment:
PR says every line in this file was replaced.  Which is odd.  Any idea why this might have happened?  I can't see what changes you made to this file.

Reply:
It looks like GitHub is misbehaving.

The only change is line 63, which is highlighted.